pkg/provisioner/manager.go

```diff
@@ -16,6 +16,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/event"
 	"sigs.k8s.io/controller-runtime/pkg/manager"
 	"sigs.k8s.io/controller-runtime/pkg/predicate"
+	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
```

Comment:
I may answer this later in the review, but is this an additional logger?

Reply:
this is the actual controller logger that we need to set if we want to get Reconile() errors reported